src/main/scala/org/broadinstitute/dsde/firecloud/service/LibraryServiceSupport.scala

```diff
@@ -21,12 +21,13 @@ trait LibraryServiceSupport {
 
   def indexableDocument(workspace: RawlsWorkspace): Document = {
     val attrfields = workspace.attributes.filter(_._1.namespace == AttributeName.libraryNamespace)
-    val idfields = Map(
+    val nonlibraryfields = Map(
       AttributeName.withDefaultNS("name") -> AttributeString(workspace.name),
       AttributeName.withDefaultNS("namespace") -> AttributeString(workspace.namespace),
-      AttributeName.withDefaultNS("workspaceId") -> AttributeString(workspace.workspaceId)
+      AttributeName.withDefaultNS("workspaceId") -> AttributeString(workspace.workspaceId),
+      AttributeName.withDefaultNS("discoverableByGroups") -> AttributeString(ElasticSearch.fieldDiscoverableByGroups)
```

Comment:
This adds:
```
"discoverableByGroups": "_discoverableByGroups",
```
to each indexed document in ES. It should instead be adding:
```
"_discoverableByGroups": ["group1","group2","group3"],
```
(note the leading underscore in field name)

BUT ... since we don't know how, yet, to retrieve the list of groups that can discover the workspace, we don't know what to populate this with. Maybe stub this out, something like:

```
    // TODO: query for groups once that feature exists
    val discoverableByGroups = Seq.empty

    val discValues = if (discoverableByGroups.nonEmpty) {
      Map(AttributeName.withDefaultNS(ElasticSearch.fieldSuggest) -> AttributeValueList(discoverableByGroups map {g => AttributeString(g)}))
    } else {
      Map.empty
    }

    Document(workspace.workspaceId, attrfields ++ nonlibraryfields ++ discValues)
```